pkg/minikube/config/config_test.go

```diff
@@ -141,3 +142,43 @@ func Test_readConfig(t *testing.T) {
 		t.Errorf("Did not read config correctly,\n\n wanted %+v, \n\n got %+v", expectedConfig, mkConfig)
 	}
 }
+
+func Test_writeConfig(t *testing.T) {
```

Comment:
I prefer that unit tests target the public method when possible, though I understand that in this case it isn't easy to do so without breaking the API.

This is definitely an improvement over what we currently have, but two alternative ideas are:

- Have the public WriteConfig function accept an argument of the file path
- Give the `config` package a var containing the default path to use that can be set by tests

Thoughts?

Reply:
If changing an IP is a possibility, let's do it. I don't like methods hiding it's internal like that.